packages/commonwealth/client/scripts/views/components/component_kit/cw_radio_button.tsx

```diff
@@ -32,29 +30,43 @@ export const CWRadioButton = (props: RadioButtonProps) => {
     checked,
     value,
   } = props;
+  const [isChecked, setIsChecked] = useState<boolean>(checked);
+
+  const handleClick = () => {
+    if (!disabled) {
+      setIsChecked(!isChecked);
+    }
+    onChange();
+  };
+
+  // For Storybook checked control
+  useEffect(() => setIsChecked(checked), [checked]);
 
-  const params = {
-    disabled,
+  const commonProps = {
```

Comment:
I should have caught this much earlier. 

2 things here:
1. we should be not modifying the components in use on the site, but creating _new_ components to be used subsequently. 
2. if we were to have merged it, it would have bricked the UI in master (see below)



Components Page Radio Buttons: 
<img width="519" alt="Screen Shot 2023-06-23 at 4 11 36 PM" src="https://github.com/hicommonwealth/commonwealth/assets/16248081/19c4f8fa-8576-4de1-a090-1328da281873">

Sidebar goes wonky after visiting pages with radio buttons (some css cascade issue introduced?): 
https://github.com/hicommonwealth/commonwealth/assets/16248081/670ede94-9955-4caf-9174-6d4156fe5a4d

Reply:
> I should have caught this much earlier.
> 
> 2 things here:
> 
> 1. we should be not modifying the components in use on the site, but creating _new_ components to be used subsequently.
> 2. if we were to have merged it, it would have bricked the UI in master (see below)
> 
> Components Page Radio Buttons: <img alt="Screen Shot 2023-06-23 at 4 11 36 PM" width="519" src="https://user-images.githubusercontent.com/16248081/248391850-19c4f8fa-8576-4de1-a090-1328da281873.png">
> 
> Sidebar goes wonky after visiting pages with radio buttons (some css cascade issue introduced?): https://github.com/hicommonwealth/commonwealth/assets/16248081/670ede94-9955-4caf-9174-6d4156fe5a4d

@zakhap fixed:

![image](https://github.com/hicommonwealth/commonwealth/assets/30223098/870c0987-6cd0-4b92-a7ec-d06222e8bd88)